### PR TITLE
Bug/33838 process v1 completed checks as default case

### PR DESCRIPTION
**Pull request by GuyHarwood**
- if no version specified on complete check message, process as v1
- increate batch size on check expiry to 1000
- double frequency of check expiry run to every 5 minutes

### CODE REVIEW

File: functions-app/poison-replay-completed-checks/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  } else {
+    version = 1
+    message = check
+  }
+  const id = uuid()
+  const file = path.join('./', `v${version}`, `${id}.json`)
+  context.log(`saving to ${file}`)
+  try {
+    const jsonString = JSON.stringify(message)
+    fs.writeFileSync(file, jsonString)
```
Comment (jon-shipley) [inline functions-app/poison-replay-completed-checks/index.js]: This seems out of place to me in a function.  If we find these messages interesting we should save them somewhere properly like a DB.

For now, surely all this function needs to do is to cycle the poison queue by injecting the messages back onto the processing queue?  Any messages that are not processed will end up back on the poison queue.

Also, why check this function in if it not working?  The write-to-file aspect suggests there might be a lot of additional functionality to come.

Comment (GuyHarwood) [reply functions-app/poison-replay-completed-checks/index.js]: this function is very much just a sketch / spike to explore peeking messages, reading them and putting back on the queue.  

Why a function?  our azure queue service library in the admin app only includes functionality for adding messages onto a queue, and given current demands i wanted the quickest route to obtain a picture of schools affected.

Ideally this should be in a completely separate PR, as to keep the conversation focused.  So i think ill do just that.

File: functions/completed-checks/v2.js [hunk 2]
```diff
@@ -32,12 +32,12 @@ const v2 = {
...
     // Default output is bound to the pupilEvents table (saved in table storage)
     context.bindings.pupilEventsTable = []
-    const base64EncodedPayload = Buffer.from(JSON.stringify(completedCheckMessage)).toString('base64')
+    // const base64EncodedPayload = Buffer.from(JSON.stringify(completedCheckMessage)).toString('base64')
     const entity = {
       PartitionKey: completedCheckMessage.checkCode,
       RowKey: uuid(),
       eventType: 'completed-check',
-      payload: base64EncodedPayload,
+      payload: 'removed',
```
Comment (GeorgeChatzigiannis) [inline functions/completed-checks/v2.js]: @GuyHarwood is the payload key-value pair really needed then?

Comment (GuyHarwood) [reply functions/completed-checks/v2.js]: not at all.  we could remove that.

Comment (GuyHarwood) [reply functions/completed-checks/v2.js]: @GeorgeChatzigiannis however, its been tested so i dont want to delay the release

File: functions/completed-checks/v1.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ const v1 = {
       PartitionKey: completedCheckMessage.checkCode,
       RowKey: uuid(),
       eventType: 'completed-check',
-      payload: completedCheckMessage,
+      payload: 'removed',
```
Comment (jon-shipley) [inline functions/completed-checks/v1.js]: 💯

File: functions/completed-checks/v2.js [hunk 4]
```diff
@@ -32,12 +32,12 @@ const v2 = {
     await v1.handleCompletedCheck(context, v1Message)
     // Default output is bound to the pupilEvents table (saved in table storage)
     context.bindings.pupilEventsTable = []
-    const base64EncodedPayload = Buffer.from(JSON.stringify(completedCheckMessage)).toString('base64')
+    // const base64EncodedPayload = Buffer.from(JSON.stringify(completedCheckMessage)).toString('base64')
```
Comment (jon-shipley) [inline functions/completed-checks/v2.js]: May as well delete commented out code

File: functions/expire-prepared-check/function.json [hunk 5]
```diff
@@ -2,10 +2,10 @@
   "disabled": false,
   "bindings": [
     {
-      "schedule": "0 */10 * * * *",
+      "schedule": "0 */5 * * * *",
```
Comment (jon-shipley) [inline functions/expire-prepared-check/function.json]: 💯

**Pull request merged**